src/pkcs/clu_pkcs8.c

```diff
@@ -0,0 +1,299 @@
+/* clu_pkcs8.c
+ *
+ * Copyright (C) 2006-2025 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+#include <wolfclu/clu_header_main.h>
+#include <wolfclu/clu_log.h>
+#include <wolfclu/clu_optargs.h>
+#include <wolfclu/pkey/clu_pkey.h>
+#include <wolfclu/x509/clu_cert.h>
+#include <wolfclu/x509/clu_parse.h>
+
+#if !defined(NO_PKCS8) && !defined(WOLFCLU_NO_FILESYSTEM)
+
+static const struct option pkcs8_options[] = {
+    {"-print_certs",  no_argument,       0, WOLFCLU_CERTFILE  },
```

Comment:
Copy past error? There is not print_certs in pkcs8.c

Reply:
Yup, fixed